src/twisted/newsfragments/11634.bugfix

```diff
@@ -0,0 +1 @@
+emailserver.tac now runs under python3.x
```

Comment:
Maybe 

```suggestion
docs/mail/examples/emailserver.tac docmentation example was updated to run under Python 3.
```

Reply:
I think this fragment actually belongs in src/twisted/mail/newsfragments